editions/tw5.com/tiddlers/widgets/Widgets.tid

```diff
@@ -1,19 +1,19 @@
 created: 20140908130500000
-modified: 20150219182745000
+modified: 20230303160747897
 tags: Concepts Reference
 title: Widgets
 type: text/vnd.tiddlywiki
 
 ~TiddlyWiki's display is driven by an underlying collection of <<.def widgets>>. These are organised into a tree structure: each widget has a parent widget and zero or more child widgets.
 
-~TiddlyWiki generates this <<.def "widget tree">> by parsing the WikiText of tiddlers. Each component of the WikiText syntax, including even the trivial case of ordinary text, generates a corresponding widget. The widget tree is an intermediate representation that is subsequently rendered into the actual display.
```

Comment:
It looks like some of the changes here are to precede "WikiText" with a tilde to prevent it being linked. It feels a bit backwards, because "WikiText" here has been deliberately written in camelcase to ensure that it creates a link. If we don't want a link then we should simple revert the camelcasing.

But in any event, none of those changes belong in a PR about fixing a JS implementation issue.